packages/react-tooltip/src/components/Tooltip/Tooltip.types.ts

```diff
@@ -23,67 +17,87 @@ export type TooltipProps = ComponentPropsCompat &
       | (React.ReactElement<React.HTMLAttributes<HTMLElement>> & { ref?: React.Ref<unknown> })
       | ((props: TooltipTriggerProps) => React.ReactNode)
       | null;
+  };
+};
 
-    /**
-     * The content displayed inside the tooltip.
-     */
-    content: ShorthandPropsCompat<ComponentPropsCompat>;
+/**
+ * Properties and state for Tooltip
+ */
+export interface TooltipCommons {
```

Comment:
```suggestion
export type TooltipCommons = {
```

Should be this `type`?

Reply:
Makes no difference in this case....